examples/java-demo/src/main/java/org/example/Main/Main.java

```diff
@@ -0,0 +1,61 @@
+package org.example.Main;
+
+import com.carta.compat.java.excel.AppendOnlySheetWriter;
+import com.carta.compat.java.exscalabur.DataCell;
+import com.carta.compat.java.exscalabur.DataRow;
+import com.carta.compat.java.exscalabur.Exscalabur;
+import com.carta.compat.java.yaml.YamlReader;
+import com.carta.yaml.YamlEntry;
+import org.apache.commons.compress.utils.Lists;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        new App().run();
+    }
+}
+
+class App {
+    public void run() {
+        final ClassLoader classLoader = this.getClass().getClassLoader();
+        final String templatePath = classLoader.getResource("excel/template.xlsx").getFile();
+        final String schemaPath = classLoader.getResource("yaml/schema.yaml").getFile();
+
+        final ArrayList<String> templates = Lists.newArrayList();
+        templates.add(templatePath);
+
+        final Map<String, YamlEntry> schemaDefinition = new YamlReader().parse(schemaPath);
+
+        final Exscalabur sword = new Exscalabur(templates, schemaDefinition, 100);
+
+        final List<DataRow> peopleData1 = Lists.newArrayList();
+        peopleData1.add(new DataRow().addCell("person", "Jon Somebody").addCell("gpa", 3.8).addCell("major", "CS"));
```

Comment:
a little nit, but since in java they prob won't be programming so functionally like this, what do you think of turning this part of the example into for loops just so its closer to what it'll probably look like in java